.mailmap

```diff
@@ -689,6 +689,7 @@ Shun Song <song.shun3@zte.com.cn>
 Shun Song <song.shun3@zte.com.cn> <root@clove83.zte.com.cn>
 Shun Song <song.shun3@zte.com.cn> <songshun134@126.com>
 Shweta Bhosale <Shweta.Bhosale1@ibm.com> <bhosaleshweta097@gmail.com>
+abhishek-kane Abhishek Kane <abhishek.kane@ibm.com> <abhishek.kane@gmail.com>
```

Comment:
It doesn't look like other entries in the `.mailmap` are including their github username at the front, so this should be removed

Reply:
Corrected. Thanks!